04-open.md

```diff
@@ -69,74 +70,132 @@ the process looks like this:
-what matters most is that:
-
-1.   every project have a file in its home directory
-     called something like `LICENSE` or `LICENSE.txt`
-     that clearly states what the license is, and
-2.   people use existing licenses rather than writing new ones.
-
-The second point is as important as the first:
-most scientists are not lawyers,
-so wording that may seem sensible to a layperson
-may have unintended gaps or consequences.
-The [Open Source Initiative](http://opensource.org/)
-maintains a list of open source licenses,
-and [tl;drLegal](http://www.tldrlegal.com/) explains many of them in plain English.
-
-When it comes to data, publications, and the like,
-scientists have many more options to choose from.
-The good news is that an organization called [Creative Commons](http://creativecommons.org/)
-has prepared a set of licenses using combinations of four basic restrictions:
-
-*   Attribution: derived works must give the original author credit for their work.
-*   No Derivatives: people may copy the work, but must pass it along unchanged.
-*   Share Alike: derivative works must license their work under the same terms as the original.
-*   Noncommercial: free use is allowed, but commercial use is not.
-
-These four restrictions are abbreviated "BY", "ND", "SA", and "NC" respectively,
-so "CC-BY-ND" means,
-"People can re-use the work both for free and commercially,
-but cannot make changes and must cite the original."
-These [short descriptions](http://creativecommons.org/licenses/)
-summarize the six CC licenses in plain language,
-and include links to their full legal formulations.
-
-There is one other important license that doesn't fit into this categorization.
-Scientists (and other people) can choose to put material in the public domain,
-which is often abbreviated "PD".
-In this case,
-anyone can do anything they want with it,
-without needing to cite the original
-or restrict further re-use.
+At the latest when a repository with source code (and/or manuscript
+and other creative writing text) becomes public, it should include a
+license, and a license disclaimer in prominent locations (such as the
+top-level README, and at the top of source code files). This is because
+as creative works, source code is automatically eligible for
+intellectual property (and thus copyright) protection. Code that appears to be, or is expressly
+advertised as freely available has *not* waived or relinquished such
+protection. Consequently, others who (re)use code that lacks a license
+statement do so enitrely on their own peril, because the author(s)
+of the software code can always unilaterally make such reuse illegal.
+
+A license solves this problem by granting rights to others (the
+licensees) that they would otherwise not have. What rights are being
+granted under which conditions differs, often only slightly, from one
+license to another. In contrast to proprietary licenses, the
+[licences](http://opensource.org/licenses/alphabetical) certified by
```

Comment:
I'd change here "[licenses](...)" to "[open licenses](...)", even if redundant, to stress the contrast between open and proprietary.